src/main/java/com/wolox/training/exception/ErrorHandler.java

```diff
@@ -18,41 +15,29 @@
 import java.util.stream.Collectors;
 
 @ControllerAdvice
-public class ErrorHandler extends ResponseEntityExceptionHandler {
+public class ErrorHandler {
 
     @ExceptionHandler({
             BookNotFoundException.class
     })
-    public ResponseEntity<Object> handleNotFound(Exception ex, WebRequest request) {
-        return handleExceptionInternal(
-                ex,
-                new Response(ex.getMessage()),
-                new HttpHeaders(),
-                HttpStatus.NOT_FOUND,
-                request
-        );
+    public ResponseEntity<Response> handleNotFound(Exception ex) {
+        return ResponseEntity.status(HttpStatus.NOT_FOUND).body(new Response(ex.getMessage()));
     }
 
     @ExceptionHandler({
             BookIdMismatchException.class,
             ConstraintViolationException.class,
             DataIntegrityViolationException.class
     })
-    public ResponseEntity<Object> handleBadRequest(Exception ex, WebRequest request) {
+    public ResponseEntity<Object> handleBadRequest(Exception ex) {
```

Comment:
This should be `ResponseEntity<Response>`, right?

Reply:
Yes, i solved in the next commit.